### ISSUE TITLE
Navigator Buttons are clickable far below the Navigator itself

### ISSUE THREAD

**Issue by perrosnk** [wix/react-native-navigation#339]
### Issue Description

The navigator button on the left is clickable till the red line, which is far below the navigator itself.
This causes a lot of problems, because in some screens I have some buttons below the navigator.
### Steps to Reproduce / Code Snippets / Screenshots

![drawing](https://cloud.githubusercontent.com/assets/10828675/18818580/a441b9b0-8385-11e6-929e-7d3715d6ca98.png)

[FILL THIS OUT - It will be extremely helpful]

---
### Environment

React Native Navigation version: 2.0.0-experimental.126
React Native version: 0.31
Platform(s) (iOS, Android, or both?): iOS
Device info (Simulator/Device? OS version? Debug/Release?): simulator iPhone 6s, iOS 10, Xcode 8

**Comment by guyca** [comment]
@perrosnk Thanks for reporting. Actually this issue has been reported a few times before already but we couldn't reproduce it. Can you reproduce this issue in the example project?

**Comment by perrosnk** [comment]
I disabled hot reloading, stopped Remote JS debugger, shutdown npm server, and then run react-native run-ios again, so the app was rebooted.
I then reactivated hot reloading and Remote JS debugger again, and it now seems to work fine. Very weird

**Comment by bdunn313** [comment]
I ran into this exact same issue. Just by eyeballing my app versus the provided image in the original description of the issue, I would say that I'm experiencing more or less the same behavior with the same extension on the clickable region.

I ended up following same steps as @perrosnk  above (disable HR, stopped remote js debugger, shutdown packager, re-ran react-native run-ios) and this worked like a charm.

If I do end up running into this again, I will try and isolate the issue further, to see if one of the above steps specifically appears to affect behavior.
